packages/workflow/src/records/fhir.ts

```diff
@@ -745,6 +954,78 @@ export function createUnassignedTask(
   return unassignedTask
 }
 
+export function createCertifiedTask(
+  previousTask: SavedTask,
+  practitioner: Practitioner
+): SavedTask {
+  return {
+    resourceType: 'Task',
+    status: 'accepted',
+    intent: 'proposal',
+    code: previousTask.code,
+    focus: previousTask.focus,
+    id: previousTask.id,
+    requester: {
+      agent: { reference: `Practitioner/${practitioner.id}` }
+    },
+    identifier: previousTask.identifier,
+    extension: [
+      ...previousTask.extension.filter((extension) =>
+        [
+          'http://opencrvs.org/specs/extension/contact-person-phone-number',
+          'http://opencrvs.org/specs/extension/informants-signature',
+          'http://opencrvs.org/specs/extension/contact-person-email'
+        ].includes(extension.url)
+      )
+    ],
```

Comment:
shouldn't all the necessary extensions be here like signatures?

Reply:
yes I'll add them in